WordPress/build.gradle

```diff
@@ -36,12 +36,19 @@ android {
         targetSdkVersion 23
 
         buildConfigField "String", "APP_PN_KEY", "\"org.wordpress.android.playstore\""
+        buildConfigField "boolean", "VISUAL_EDITOR_AVAILABLE", "false"
     }
 
     productFlavors {
-        vanilla {}
+        vanilla {} // used for release and beta
 
-        zbetagroup {
+        zalpha { // alpha version - enable experimental features
+            buildConfigField "boolean", "VISUAL_EDITOR_AVAILABLE", "true"
+            applicationId "org.wordpress.android"
```

Comment:
This has the same `applicationId "org.wordpress.android"` of the "vanilla" version of WP. Is that intended? Can't install the 2 versions side by side?